.github/workflows/publish.yaml

```diff
@@ -80,29 +88,25 @@ jobs:
       - name: Create GitHub release
         id: gh_release
         run: |
-          # Get the version file that we updated in the previous job
-          wget https://raw.githubusercontent.com/ersilia-os/ersilia/master/ersilia/_static_version.py
-        
           month=$(date +'%B')
           year=$(date +'%Y')
           title="$month $year"
-          version=`sed -n 's/.*version = "\([^"]*\)".*/\1/p' _static_version.py`
+          version=${{ needs.version.outputs.VERSION }}
```

Comment:
Are we sure this will work? I am not very sure with passing variables between jobs in a GH actions pipeline, and I don't want this failing next month so I'll just leave this as is. :/

Reply:
I have left the `sed` command for version extraction from `_static_version.py` unchanged you suggested. But here is my recommendation:
 In the version job, we are extracting the version using the static_version_writer.py script and we are also using `sed` to download the `_static_version.py`  again. This can cause redundancy, we can consider using the version extracted in the `version` job and pass it as an output throughout the pipeline.